parser/name_parser.go

```diff
@@ -7,13 +7,15 @@ import (
 	"github.com/image-server/image-server/core"
 )
 
-var reR, reS, reW, reF *regexp.Regexp
+var reR, reS, reW, reF, reFormat *regexp.Regexp
 
 func init() {
-	reR = regexp.MustCompile(`^([0-9]+)x([0-9]+)(?:-q([0-9]+))?\.(\w{3,4})$`)
```

Comment:
this regexp did not catch extensions like `DEBUG` which is valid for image magic.